### PR TITLE
Updated user sign-in/login and creation of user-session #17

### LINKED ISSUE #12: User Sign Up Page
1. Build view
2. API to communicate with back end/database

### LINKED ISSUE #17: Task: After sign in, create a login session and dashboard
Users must log in, get authenticated, get redirected to a home page/dashboard, and then be given a login session.

### PR DESCRIPTION
**Pull request by gregtuc**
Several updates. Manual POST method in clients login.js was replaced with methods from Axios, allowing for an easier process. After signing in, session is now created with express-session - this will allow the user to remain signed in (see path /login in server index.js) which should enhance #17 , though additional features are needed. Basic sign-in page was also complete which closes #12 .
Very basic layout for a home page was also created, mostly for the purposes of future testing of redirection after sign-in.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,57 +1,80 @@
...
 
 //Mongo Atlas requirements
-const mongoclient = require('mongodb').MongoClient;
-const objectid = require('mongodb').ObjectID;
-const uri = "mongodb+srv://<username>:<password>@cluster0-pf0b2.mongodb.net/test?retryWrites=true&w=majority";
-const userModel = require('./user-schema');
+const mongoclient = require("mongodb").MongoClient;
+const objectid = require("mongodb").ObjectID;
+const uri =
+  "mongodb+srv://greg34910:cookie123@cluster0-pf0b2.mongodb.net/test?retryWrites=true&w=majority";
```
Comment (ginywiny) [inline server/index.js]: You are adding your personal mongoDB account info. Can you make this generic?

Comment (gregtuc) [reply server/index.js]: Oops! Nice catch, will change that.

**Comment by ginywiny** [comment]
Don't forget to reference commits with issue numbers, so we can reference the issue if ever there are multiple issues worked on for one pull request

**Comment by gregtuc** [comment]
> Don't forget to reference commits with issue numbers, so we can reference the issue if ever there are multiple issues worked on for one pull request

Two separate issues should already be referenced in the description

**Pull request merged**